### PR TITLE
[vslib]: MACsec in Gearbox

### PR DESCRIPTION
**Pull request by Pterosaur**
Add the placeholders of MACsec SAI in Gearbox BCM81724.

We don't really implement the MACsec in Gearbox but just make all MACsec related APIs in SAI that can be called successfully in SWSS.

### CODE REVIEW

File: vslib/SwitchBCM81724.h [hunk 1]
```diff
@@ -29,6 +29,31 @@ namespace saivs
...
+
+            virtual sai_status_t remove(
+                    _In_ sai_object_type_t object_type,
+                    _In_ const std::string &serializedObjectId);
+
+            virtual sai_status_t set(
+                    _In_ sai_object_type_t objectType,
+                    _In_ const std::string &serializedObjectId,
+                    _In_ const sai_attribute_t* attr);
+
```
[the remark this answers is not in the document]
Comment (Pterosaur) [reply vslib/SwitchBCM81724.h]: Done, please check.

**Comment by judyjoseph** [comment]
@kcudnik Could you take a look again - approve if fine?

**Review by kcudnik — APPROVED** [review]

**Comment by rlhui** [comment]
@pterosaur - only lgtm is failing now

**Pull request merged**